planetary-variables/soil-water-content/soil-water-content-anomaly/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Soil Water Content Anomaly
+grand_parent: Planetary Variables
+parent: Soil Water Content
+layout: script
+nav_exclude: false
+scripts:
+    - [Visualization, script.js]
+    - [Raw Values, raw.js]
+---
+
+## Description of representative images
+
+The visualization represents less water content than normal as red and more water than normal as blue.
+
+Soil Water Content Anomaly (C band 1000 m) on June 3rd, 2023 Graz, Austria.
+
+| True Color Image of AOI (June 9th) |      Soil Water Content (SWC)      |              Standardized Anomaly of SWC               |
+| :--------------------------------: | :--------------------------------: | :----------------------------------------------------: |
+| ![True Color](fig/true_color.png)  | ![Soil Water Content](fig/swc.png) | ![Soil Water Content Anomaly example](fig/anomaly.png) |
+
+## General description
+
+This script calculates the standardized anomaly of the soil water content for a particular date. It takes all values of the same day of the year in previous years and calculates the mean and standard deviation of the value. The anomaly is then defined as the current value subtracted by the mean of the reference period. To get the _standardized_ anomaly, the absolute anomaly value is then divided by the standard deviation of the reference period.
+
+The standardized anomaly can be compared between different areas and different sensors. If the absolute anomaly is desired the last step of dividing by the standard deviation can be omitted. This results in anomalies in the unit of measurement. In this case $$m^3/m^3$$ below or above the mean water content during the reference period.
+
+## Notes on usage
+
+### EO Browser
+
+To use this script in the EO Browser, a time span needs to be set in the interface. To do this, visualize the date you want to calculate an anomaly for. Then in the Visualize panel, hit the green `Timespan` button.
+
+![Visualize Panel Interface](fig/single_date.png)
+
+In the interface which then appears, select the time range you want to use as reference period. In this case, we select a time range from 2012 to 2024, which is 12 years. Be aware that this will only include data which is available, so if you ordered data for 5 years but specify a time range of 10 years, only the 5 years you have ordered will be included.
+
+The date that will be compared to the reference period is always the most recent date containing data in the selected time span.
```

Comment:
Suggestion:

_Please note_: The date that is compared to the reference period is always the most recent date with data in the selected time span.